web/src/app/UserTasks.tsx

```diff
@@ -11,14 +11,15 @@ export default function UserTasks() {
   if (!isAuthed) return <LoginFallback />;
 
   return (
-    <div className="card">
+    <div className="card card--user-tasks">
```

Comment:
it doesn't look like the styles have been defined anywhere for either `card` or `card--user-tasks`. I'd go through and delete any css that is not currently being used.